binutils/binutils-2.26.def

```diff
@@ -0,0 +1,86 @@
+# Author: Paulo Souza, Shahzeb Siddiqui
+
+# docker-bootstrap:
+# Copyright (c) 2015-2016, Gregory M. Kurtzer. All rights reserved.
+# 
+# "Singularity" Copyright (c) 2016, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory (subject to receipt of any
+# required approvals from the U.S. Dept. of Energy).  All rights reserved.
+
+BootStrap: docker
+From: centos
+Include: yum
+
+
+# eb-gcc:
+%post
+		export EASYBUILD_INSTALLPATH=/app
+
+		# scratch space to build and download source files from eb
+		export EASYBUILD_PREFIX=/scratch
+		mkdir -p $EASYBUILD_PREFIX
+		chmod 777 $EASYBUILD_PREFIX
+
+                mkdir -p /app
+                yum groupinstall -y "Development Tools"
+ 
+                # install csh/tcsh shell
+                yum install tcsh -y
+ 
+                # epel.repo needed for Lmod
+                yum install -y epel-release
+                yum install -y Lmod
+                # vi 
+                yum install -y vim-enhanced
+
+                # References:
+                # https://stackoverflow.com/questions/36327805/how-to-install-gcc-5-3-with-yum-on-centos-7-2
+                # https://gcc.gnu.org/wiki/InstallingGCC
+                yum install -y libmpc-devel mpfr-devel gmp-devel
+                yum install -y zlib-devel*
+                yum install -y curl
+ 
+                # eb dependencies
+                yum install python-devel -y
+                yum install python-setuptools -y
+
+                yum install -y git 
+                export LC_ALL=C
+                yum install -y python-pip
+                pip install --upgrade pip
+                pip install -U setuptools
+                pip install -U easybuild
+                yum install -y git
+                chmod 777 /app
+                useradd singularity
+                . /etc/profile
+                module load lmod
```

Comment:
I agree, I don't think we need this, also no need to load lmod module. I think runuser singularity -c <command> will run in Interactive Non-login mode. With bash we need the singularity user to have ~/.bashrc which I confirmed did exist using useradd command. There is a known issue  with Interactive Non-login for Bash user in Lmod see page 26 of https://media.readthedocs.org/pdf/lmod/latest/lmod.pdf. The solution seems to be in implemented already so singularity should pick up module command with this shell invocation.